docs/content/dagster-cloud/managing-deployments/alerts/managing-alerts-in-ui.mdx

```diff
@@ -0,0 +1,173 @@
+---
+title: "Managing alert policies in Dagster Cloud | Dagster Docs"
+description: "Create and manage alert policies using the Dagster UI."
+
+platform_type: "cloud"
+feature: "alerting"
+---
+
+# Managing alert policies in Dagster Cloud
+
+<ApplicableDagsterProduct />
+
+In this guide, we'll walk you through managing alert policies in Dagster Cloud.
+
+**Prefer to work in code?** You can also define and set alert policies using the [`dagster-cloud` CLI](/dagster-cloud/managing-deployments/alerts/managing-alerts-cli).
+
+---
+
+## Prerequisites
+
+To follow the steps in this guide, you'll need:
+
+- **Organization, Admin, or Editor permissions in Dagster Cloud**, which are required to create and manage alerts
+- **Optional**: If using an integration like [Microsoft Teams](/dagster-cloud/managing-deployments/alerts/microsoft-teams) or [Slack](/dagster-cloud/managing-deployments/alerts/slack) as a notification service, complete the integration setup before proceeding. Refer to the [Supported notification services](/dagster-cloud/managing-deployments/alerts#supported-notification-services) for a complete list of integrations.
+
+---
+
+## Creating alert policies
```

Comment:
if the bulk editing gets screenshots, should this get them too?

Reply:
I don't feel that it's really necessary, since the path to creating a policy is more obvious than bulk editing since it's hidden under a not-labeled menu